src/ObjectList.php

```diff
@@ -519,6 +519,15 @@ public function countMatching(callable $filter)
     }
 
     /**
+     * @param callable $filter
+     * @return bool
+     */
+    public function hasMatching(callable $filter): bool
+    {
+        return $this->countMatching($filter) > 0;
```

Comment:
I've seen this pattern before and I prefer to do `$this->find($filter) !== null` as it's performance is significantly better on large collections